docs/api/base-window.md

```diff
@@ -144,10 +144,27 @@ _**Note**: There is a subtle difference between the behaviors of `window.onbefor
 Emitted when the window is closed. After you have received this event you should
 remove the reference to the window and avoid using it any more.
 
+#### Event: 'query-session-end' _Windows_
+
+Returns:
+
+* `event` Event
+* `reasons` string[]
+
+Emitted when window session is going to end due to force shutdown or machine
```

Comment:
This mentions "force shutdown". Does that mean `query-session-end` will fire again if the user elects to force a shutdown in the "These apps are preventing shutdown" screen? I would have intuitively expected that to trigger the `session-end` event instead.

Reply:
No, it won't I will remove "force" word.